### PR TITLE
toast notifications

### PR DESCRIPTION
**Pull request by donsalari**
o1

toast notifications disappears after 5 seconds.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,7 +36,8 @@
     "electrode-redux-router-engine": "^1.0.0",
     "electrode-server": "^1.0.0",
     "electrode-static-paths": "^1.0.0",
-    "lodash": "^4.10.1"
+    "lodash": "^4.10.1",
+    "react-notify-toast": "^0.1.3"
```
Comment (jchip) [inline package.json]: is it required in prod?

Comment (donsalari) [inline package.json]: @ananavati i have moved to `dep` instead of `dev`: https://github.com/electrode-io/electrode-archetype-react-app/pull/62

Comment (ananavati) [inline package.json]: yep its needed for prod

File: config/sw-config.js [hunk 2]
```diff
@@ -9,9 +9,8 @@ module.exports = {
       handler: "fastest",
       urlPattern: "\/$"
     }],
-    // context for script paths is the client folder
-    importScripts: ['./sw.js']
-    },
+    importScripts: ['./sw-events.js']
```
Comment (ananavati) [inline config/sw-config.js]: any reason to rename from `sw` to `sw-events` ?

Comment (donsalari) [inline config/sw-config.js]: `sw-registration` will do all the sw initialization, registration and handle states related to sw bootstrapping. 
`sw-events` will contain all the user defined event like push-notifications/background sync.

**Pull request merged**